tests/QueueReceiver.Core.UnitTests/Services/PersonProjectHistoryHelperTests.cs

```diff
@@ -83,7 +83,7 @@ public void LogUnvoidProjects()
             var personProjectHistoryOperations = personProjectHistory.PersonProjectHistoryOperations.FirstOrDefault();
 
             Assert.IsTrue(personProjectHistory.PersonProjectHistoryOperations.Count == 1);
-            Assert.IsTrue(personProjectHistoryOperations.UpdatedByUser == PersonProjectHistoryConstants.UpdatedBy);
+            Assert.IsTrue(personProjectHistoryOperations.UpdatedByUser == "ACCESS_SYNC");
```

Comment:
Think we can just remove this line, as updatedByUser is no longer handled by this class. Should be tested in test class for PersonProjectHistoryRepository.

Reply:
DbContextSettings rewritten to PersonCreatedByCache